From 56adf0cb25422c1bc4f64313d3a90fa700707dda Mon Sep 17 00:00:00 2001
From: Jarkko Nikula <jhnikula@gmail.com>
Date: Mon, 18 Apr 2011 19:21:31 +0000
Subject: [PATCH 0324/1092] cbus: retu: Fix build breakage caused by recent
 genirq commits

Convert retu.c to use updated irq function names after the genirq cleanup
commits 0c6f8a8 "genirq: Remove compat code" and
a0cd9ca "genirq: Namespace cleanup".

Signed-off-by: Jarkko Nikula <jhnikula@gmail.com>
Signed-off-by: Tony Lindgren <tony@atomide.com>
---
 drivers/cbus/retu.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/cbus/retu.c b/drivers/cbus/retu.c
index f271e57..ec108f3 100644
--- a/drivers/cbus/retu.c
+++ b/drivers/cbus/retu.c
@@ -298,10 +298,10 @@ static void retu_irq_init(struct retu *retu)
 	int			irq;
 
 	for (irq = base; irq < end; irq++) {
-		set_irq_chip_data(irq, retu);
-		set_irq_chip_and_handler(irq, &retu_irq_chip,
+		irq_set_chip_data(irq, retu);
+		irq_set_chip_and_handler(irq, &retu_irq_chip,
 				handle_simple_irq);
-		set_irq_nested_thread(irq, 1);
+		irq_set_nested_thread(irq, 1);
 		retu_irq_setup(irq);
 	}
 }
@@ -316,8 +316,8 @@ static void retu_irq_exit(struct retu *retu)
 #ifdef CONFIG_ARM
 		set_irq_flags(irq, 0);
 #endif
-		set_irq_chip_and_handler(irq, NULL, NULL);
-		set_irq_chip_data(irq, NULL);
+		irq_set_chip_and_handler(irq, NULL, NULL);
+		irq_set_chip_data(irq, NULL);
 	}
 }
 
@@ -479,7 +479,7 @@ static int __init retu_probe(struct platform_device *pdev)
 		goto err1;
 	}
 
-	set_irq_wake(retu->irq, 1);
+	irq_set_irq_wake(retu->irq, 1);
 
 	/* Register power off function */
 	pm_power_off = retu_power_off;
-- 
1.7.11.2

